DBSCE/Overview.md

```diff
@@ -0,0 +1,219 @@
+
+DBSC(E) (in contrast with DBSC):
+
+![DBSC(E) Highlevel Design](<./DBSC(E).svg>)
+
+Highlights:
+
+Note: All references to RP, IDP are equivalent to `server` in the original [DBSC design](https://githuub.com/wicg/dbsc).
+
+1. **Pre-Session initiation with special headers (steps 1-2):** When a user starts a sign-in process, or initiates a session, the webpage initiating the session sends special headers `Sec-Session-GenerateKey` and `Sec-Session-HelperIdList` to the browser in response, to indicate that the session is expected to be DBSC(E) compliant.
+
+   - The `Sec-Session-GenerateKey` header contains the URL of the server(RP), the URL of the IdP (authentication service in most cases - which is optional for consumer use cases), a `nonce`(challenge) and any extra parameters that the IdP wants to send to the Local Key Helper.
+   - As the Local Key Helper is public or thirdparty, and potentially can be from a different vendor, `nonce`(challenge) is essential to prevent the clock-skew between the IdP and the Local Key Helper, especially for short lived Binding statements. `nonce`(challenge) is expected to be generated by the IdP/RP as a part of the request, a random number with time sensitivity, and is expected to be embedded in the binding statement. IdP will be able to validate nonce to ensure the binding statement is freshly issued.
+   - The `Sec-Session-HelperIdList` header contains a list of helper IDs that the browser can use to generate the key. As we touched upon before, there could be multiple devices on a single physical device, and/or multiple device registration clients on a single device. The `HelperId` helps the browser to choose the right **Local Key Helper** to generate the key. The browser will evaluate the policy for the IdP and the helper IDs, and choose the appropriate helper ID to generate the key. The browser will then call the Local Key Helper to generate the key.
+
+1. **Key and Binding Statement Generation (steps 3-7):** The Local Key Helper generates the key and the binding statement. AIK refers to the `Attestation Key` described [above](#attestation-key). The binding statement is expected to contain the `nonce`(challenge) sent by the IdP, the thumbprint of the public key, and any extra claims that the IdP wants to send.
```

Comment:
Attestation key section - should it be attestation service section instead?

Reply:
Added `Attestation Key` now.